libs/blocks/quiz-entry/quiz-entry.css

```diff
@@ -248,26 +361,29 @@ html[dir="rtl"] .quiz-option-icon {
   align-items: center;
   display: flex;
   justify-content: center;
-  margin: 50px 0 24px;
 }
 
 .quiz-button {
   background: var(--color-gray-800);
-  border: 2px solid var(--color-white);
-  border-radius: 30px;
+  border: none;
+  border-radius: 20px;
   cursor: pointer;
   font-family: inherit;
-  font-size: 14px;
+  font-size: 17px;
   font-weight: 700;
-  min-height: 32px;
-  min-width: 100%;
-  padding: 11px 86px;
+  line-height: 21px;
+  padding: 9px 16px;
+}
+
+.quiz-button:focus-visible {
+  outline: 4px solid #4B75FF;
+  outline-offset: 2px;
 }
 
 .quiz-button-label {
   color: var(--color-white);
   font-size: 18px;
```

Comment:
font-size: 17px
line-height: 21px

Reply:
again, this is a spot where standard milo / consonant values make sense.